ItunesSearch/ItunesSearch/SearchResultController.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  SearchResultController.swift
+//  ItunesSearch
+//
+//  Created by Keri Levesque on 2/11/20.
+//  Copyright © 2020 Keri Levesque. All rights reserved.
+//
+
+import Foundation
+
+class SearchResultController {
+    let baseURL = URL(string: "https://itunes.apple.com/search")! // base URL for the itunes search API
+    
+    var searchResults: [SearchResult] = [] // data source for the table view
+
+    enum HTTPMethod: String {
+        case get = "GET"
+        case put = "PUT"
+        case post = "POST"
+        case delete = "DELETE"
+    }
+    func performSearch(searchTerm: String, resultType: ResultType, completion: @escaping (_ error: Error?) -> Void) {
```

Comment:
In the closure you want to put just `Error?` That's why you are getting an error in your TableViewController.
The` underscore error:` acts as a variable.
This is how you want it to look:
`func performSearch(searchTerm: String, resultType: ResultType, completion: @escaping (Error?) -> Void) {`